app/services/analytics_events.rb

```diff
@@ -565,6 +565,16 @@ def idv_verify_in_person_troubleshooting_option_clicked(flow_path:, **extra)
     )
   end
 
+  # @param [Hash] variant The IPP CTA Variant that was shown to the user
+  # The user was shown a variant of the IPP CTA
+  def idv_ipp_cta_variant(variant:, **extra)
+    preserve_testing_value({ ipp_cta_variant: variant })
+    track_event(
+      'IdV: IPP CTA Variant Displayed: ' + variant.to_s,
+      **extra,
+    )
```

Comment:
The purpose of this file is to have singuar static strings per event name so we can document them easily.

When we used to do more dynamic interpolation, it was difficult to find and aggregate across the variants, it would be much more effective to have a static string  and a separate explicit `variant:` property so that we can filter on these in Cloudwatch, I would strongly recommend changing this to:

```ruby
    track_event(
      'IdV: IPP CTA Variant Displayed',
      variant: variant,
      **extra,
    )
```